packages/components/src/editing/EditorPane.tsx

```diff
@@ -41,6 +41,14 @@ export default function EditorPane({
   const monaco = useMonaco();
   const editorRef = useRef<editor.IStandaloneCodeEditor | null>(null);
   const statusBarRef = useRef<HTMLDivElement>(null);
+
+  if (monaco != null && onWrite != null) {
```

Comment:
We generally prefer strict inequality (`===` and `!==`) in all cases. See [MDN docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness) for details.

Reply:
for some reason, `onWrite !== null` throws an error 
![image](https://user-images.githubusercontent.com/45471522/219142304-5a700bd7-702a-436f-9b02-dc80ed33f9ea.png)
but `onWrite != null` doesn't
![image](https://user-images.githubusercontent.com/45471522/219142417-e50e9fe1-d473-4c54-a86b-1e19ce5b1835.png)
and I'm not sure why